src/main/resources/reference.conf

```diff
@@ -134,4 +134,40 @@ cassandra-snapshot-store {
       parallelism-max = 8
     }
   }
+}
+
+cassandra-query-journal {
```

Comment:
I think we want the config to be separated from write journal to make the two components completely independent.  Happy to merge (the duplicated keys) if you think it would be better.